client/components/data/query-inline-help-search/index.jsx

```diff
@@ -24,7 +24,7 @@ class QueryInlineHelpSearch extends Component {
 	}
 
 	request( props ) {
-		if ( props.requesting || ! props.query ) {
+		if ( props.requesting ) {
```

Comment:
How come it's safe to remove this now?

Reply:
Because the request action deals when the query is empty. It could be confusing but the best way to understand it (it works to me :-D) is keeping in mind that event when the query is empty, it will return the default results list.

```es6
export function requestInlineHelpSearchResults( searchQuery = '' ) {
	return ( dispatch, getState ) => {
		// ...
		// If the search is empty return contextual results and exist
		// early to avoid unwanted network requests.
		if ( ! searchQuery ) {
			dispatch( {
				type: INLINE_HELP_SEARCH_RESET,
				searchResults: contextualResults,
			} );

			// Exit early
			return;
		}
	// ...
```

If not, it's going to show an empty results box:

<img src="https://user-images.githubusercontent.com/77539/86114975-ab05b380-baa1-11ea-9a87-0dc910e051fd.png" width="400px" />

Reply:
Yep ok that makes sense.